contracts/access/access-control/impls.rs

```diff
@@ -55,6 +55,15 @@ pub trait AccessControl: AccessControlStorage {
         self._has_role(&role, &address)
     }
 
+    /// The user must override this function using their event definition.
+    fn emit_role_admin_changed(&mut self, role: RoleType, previous_admin_role: RoleType, new_admin_role: RoleType);
```

Comment:
We decide to not force the user to implement it. So let's define empty implementation instead of pure function